src/components/Link.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Link = ({ place, onClick }) => (
+const Link = ({ place, onClick, id }) => (
   <a
     className="dropdown__list__item"
     onClick={onClick}
```

Comment:
I suggest to just pass all the necessary data to a event handler function and omit data attributes: 
`(event) => onClick({ event, palce, id } )`